packages/utils/lib/schema.js

```diff
@@ -7,6 +7,16 @@ function overridableValue (spec, defaultValue) {
   }
 }
 
+function removeDefaults (schema) {
+  const cloned = structuredClone(schema)
+
+  for (const value of Object.values(cloned.properties)) {
+    delete value.default
+  }
+
+  return cloned
+}
+
```

Comment:
why are you removing defaults?

Reply:
I need in the `@platformatic/runtime` in the `mappings` section, otherwise merging of options from the root, the `services` section and the `mappings` section will not be possible.